fieldservice_sale/models/sale_order.py

```diff
@@ -89,7 +89,7 @@ def _prepare_fsm_values(self, **kwargs):
             "location_id": self.fsm_location_id.id,
             "location_directions": self.fsm_location_id.direction,
             "request_early": self.expected_date,
-            "scheduled_date_start": self.expected_date,
+            "scheduled_date_start": self.commitment_date,
```

Comment:
Just like with my other comment, I'd align to core here: https://github.com/odoo/odoo/blob/66c5d10833af8946ab5d3f887086f9ba08d36ab3/addons/sale_stock/models/sale_order_line.py#L260

```suggestion
            "scheduled_date_start": self.commitment_date or self.expected_date,
```